hw7/searchTree/searchTree.c

```diff
@@ -0,0 +1,141 @@
+﻿#define _CRT_SECURE_NO_WARNINGS
+
+#include <stdio.h>
+#include <string.h>
+#include "tree.h"
+
+#define ARRAY_SIZE 50
+
+int menu(Tree** tree)
```

Comment:
```suggestion
int menu(Tree* tree)
```
Вы нигде его адрес не используете вроде